src/modules/Common/data-components/Username.tsx

```diff
@@ -74,11 +87,26 @@ const DataUsername = ({
         exportName={exportName}
       />
       <div className="fr-mt-8w">
-        <BotProbabilityGraph
+        <BarGraph
+          title="Bot probability"
+          subtitle="In number of users, views on bar chart"
           data={botRepartition.map((value: number, index: number) => ({
             x: index,
             y: value,
           }))}
+          xAxis={{
+            categories: [...Array(botRepartition.length)].map((_, index) => `${index}%`),
+          }}
+        />
+      </div>
+      <div className="fr-mt-8w">
+        <BarGraph
+          title="Nb of tweets per user"
```

Comment:
~~Nb of tweets per user~~ -> `Tweets per user`
~~In number of users, views on bar chart~~-> `Views on bar chart`